src/domain/community/inviteContributors/VirtualContributors.graphql

```diff
@@ -0,0 +1,94 @@
+query AvailableVirtualContributors(
+  $filterSpace: Boolean = false
+  $filterSpaceId: UUID = "00000000-0000-0000-0000-000000000000"
+) {
+  lookup @include(if: $filterSpace) {
+    space(ID: $filterSpaceId) {
+      id
+      community {
+        id
+        roleSet {
+          id
+          virtualContributorsInRole(role: MEMBER) {
+            ...VirtualContributorFull
+          }
+        }
+      }
+      account {
+        id
+        virtualContributors {
+          ...VirtualContributorFull
+        }
+      }
+    }
+  }
+  virtualContributors @skip(if: $filterSpace) {
+    ...VirtualContributorFull
+  }
+}
+
+query AvailableVirtualContributorsInLibrary {
+  platform {
+    id
+    library {
+      id
+      virtualContributors {
+        searchVisibility
+        ...VirtualContributorFull
+      }
+    }
+  }
+}
+
+query CommunityVirtualMembersList(
+  $roleSetId: UUID!
+  $spaceId: UUID = "00000000-0000-0000-0000-000000000000"
+  $includeSpaceHost: Boolean = false
+) {
+  lookup {
+    space(ID: $spaceId) @include(if: $includeSpaceHost) {
+      provider {
```

Comment:
missing `space.id`

Reply:
It's copied from CommunityMembersList and it's the same there. I'll add it on both places.